apps/flow.md

```diff
@@ -0,0 +1,46 @@
+# Flow
+How to be Good at Flow
+
+## Terms
+
+### Workspaces
+Workspaces contain all of the info for large projects where clients and contractors are allowed to see our process in action. These are the top level of the Flow hierarchy.
+
+### Projects
+Projects contain all of the info for a subset of a project in a workspace. For Good, this is how we split our internal projects from each other. For larger projects, this will typically be an Active Sprint project and a Backlog project.
+
+### Sections
+Sections split out projects into logic sections for things like To-Do, In Progress, etc.
+
+### Tasks
+Tasks are the smallest unit of the workflow and are moved in between sections and sometimes projects.
+
+### Tags
+Tags should be used to categorise tasks in a focused manner. Try not to exceed 3 tags on one task. Here are the common tags:
+
+* pm - project management
+* fe - front end
+* be - backend
+* fs - full stack
+* de - design
+* ux - user experience
+* ui - user interface
+* bd - business development
+* feature
+* bug
+* debt
+
+## Using Flow
+
+### Managing Tasks
+Tasks should represent a single thing that needs to be completed and can be accomplished within the span of a day. If a task spans multiple days, it needs to be broken down. Make sure you don't break down tasks too much. Twenty tasks to accomplish something that only takes an hour is obviously excessive. Tasks should be tagged and dated appropriately. If you know someone cares about the progress of a task, subscribe them to it. For lengthier tasks that don't need to be broken down, use subtasks so any subscribers have an idea of the progress that's been made.
```

Comment:
I'd like to do a more granular breakdown of the tasks section since it's so important.

### Tasks

#### Naming Tasks
Task names should be concise enough for a quick glance, but not so ambiguous that only you know what the deliverable for that task is.

#### Notes & Files for Tasks
Perhaps one of the best and most important sections to utilize when creating a task. Leave any notes and links relevant to the task. Attach files that will be relevant to the task.

**Empathize with the team here.** When leaving notes, ensure your notes contain all the highlights and details for this task so that your teammates (or your future self) can quickly access all details and resources needed for the task as well easily get a sense of what needs to be accomplished. In addition to having the right information, also ensure you are sharing _readable information_. Always strive for readability — avoid huge blobs of text. Use good formatting sense. This ultimately saves time for anyone involved with the task, even if it's only you.

You can use the rich-text formatting tools in Flow or you can enable Markdown formatting in your Flow preferences. Whatever Flowts your boat.

#### Workspace

#### Project

#### Due Date

#### Assignee

#### Subscribers

#### Tags

#### Subtasks

#### Comments



^ I'd like to take an approach like this. Also, now after typing out the stuff for notes & files, it actually may be better to make a Communication/Language/Resources guidelines doc and say both notes & comments should adhere to our Communication guidelines. That way, we can have one document that standardizes how we supply information/project details to each other and just point to that doc from anywhere else in our guides.